app/views/spree/admin/payments/new.html.haml

```diff
@@ -1,4 +1,4 @@
-= render partial: 'spree/admin/shared/order_tabs', locals: { current: 'Payments' }
+= render :partial => 'spree/admin/shared/order_page_title', locals: { current: 'Payments' }
 
 - content_for :page_title do
```

Comment:
doesn't this conflict with the order_page_title partial?

Reply:
no, it adds to it, content_for appends content.